bitmind/validator/forward.py

```diff
@@ -62,10 +62,12 @@ async def forward(self):
         label = 0
         real_dataset_index, source_dataset = sample_dataset_index_name(self.real_image_datasets)
         real_dataset = self.real_image_datasets[real_dataset_index]
-        sample = real_dataset.sample(k=1)[0][0]  # {'image': PIL Image ,'id': int}
+        samples, idx = real_dataset.sample(k=1)  # {'image': PIL Image ,'id': int}
+        sample = samples[0]
 
         wandb_data['dataset'] = source_dataset
-        wandb_data['image_index'] = sample['id']
+        wandb_data['image_index'] = idx[0]
+        wandb_data['image_id'] = sample['id']
```

Comment:
Suggestion/Question, Non-blocking: Do you think renaming "image_id" to "image_name" would improve clarity? I get "index" and "id" mixed up occasionally, but maybe that's just me.

Reply:
Yeah good point, agree that's a better name. I'll update this now.